domain/src/test/java/org/oppia/android/domain/topic/TopicListControllerTest.kt

```diff
@@ -425,32 +482,26 @@ class TopicListControllerTest {
       RATIOS_TOPIC_ID,
       RATIOS_STORY_ID_1,
       RATIOS_EXPLORATION_ID_2,
-      getCurrentTimestamp()
+      getOldTimestamp()
```

Comment:
Why this change?

Reply:
Test function name is to show all older stories.